File: 프로그래머스/lv1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.py
def solution(n):
    reversed_n = str(n)[::-1]       # 정수 n을 문자열로 변환, 역순으로 정렬
    answer = []
    for r in reversed_n:            # 역순으로 정렬했던 String 상태의 n을 하나씩 불러옴
        answer += [int(r)]          # int로 재변환해서 더함
    return answer

File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.java
import java.util.List;

class Solution {
    public int[] solution(long n) {
        String s = n + "";
        int[] answer = new int[s.length()];
        int i = 0;
        while (n > 0) {
            answer[i++] = (int)(n % 10);
            n /= 10;
        }
        return answer;
    }
}